app/routes/application.js

```diff
@@ -63,15 +67,18 @@ export default Route.extend(ApplicationRouteMixin, {
     if (!this.get('session.skipRedirectOnInvalidation')) {
       this._super(...arguments);
     }
+
     this.set('session.skipRedirectOnInvalidation', false);
   },
 
-  sessionAuthenticated() {
-    if (this.get('session.previousRouteName')) {
-      this.transitionTo(this.get('session.previousRouteName'));
```

Comment:
Where's this logic gone?

Reply:
This was not working before or now, Do you think it is needed, as it does not redirect to the previous route?

Reply:
I have fixed that in the WIP PR

Reply:
@iamareebjamal I think that WIP wont be needed any more, Changing 1-2 lines fixes the issues .

Reply:
So now it redirects to call for speakers? This is what the WIP PR is for. To redirect to correct endpoint after login

Reply:
> crashes

That bug is fixed with the following service and user authentication approach used in here

Reply:
>  Test for that in fastboot mode as well

Sure checking